### PR TITLE
Add SLANDLES test for 'can parse a recipe with slot constraints on verbs'

### PR DESCRIPTION
**Pull request by Cypher1**

### CODE REVIEW

**Comment by Cypher1** [comment]
Not sure what is failing in Travis and Appveyor. I can't find the failing tests.

**Comment by shans** [comment]
it's in lint

```
/home/travis/build/PolymerLabs/arcs/runtime/test/manifest-test.js
  1735:46  error  Unnecessary escape character: \`  no-useless-escape
```

File: runtime/test/manifest-test.js [hunk 1]
```diff
@@ -1705,6 +1705,26 @@ resource SomeName
...
+        &verb
+          foo consume provideSlot
+    `);
+
+    const recipe = manifest.recipes[0];
+
+    assert.equal(recipe.particles[0]._verbs[0], 'verb');
+    assert.isUndefined(recipe.particles[0]._spec);
+    const slotConnection = recipe.particles[0].connections.foo;
+    assert.equal(slotConnection._direction, '\`consume');
```
Comment (shans) [inline runtime/test/manifest-test.js]:
```suggestion
    assert.equal(slotConnection._direction, '`consume');
```

**Comment by Cypher1** [comment]
Checking that I have managed to associate my email with my GitHub user name in the CLA bot's data store.

**Comment by Cypher1** [comment]
Thanks Shane

**Pull request merged**